internal/services/providers/gke/gke_test.go

```diff
@@ -19,15 +21,19 @@ import (
 )
 
 func TestProvider_RegisterCluster(t *testing.T) {
```

Comment:
Could you also add a test for failed autodiscovery case?

Reply:
Done